dataactvalidator/filestreaming/csvAbstractReader.py

```diff
@@ -19,10 +19,12 @@ class CsvAbstractReader(object):
     def openFile(self,region,bucket,filename,csvSchema,bucketName,errorFilename):
         """ Opens file and prepares to read each record, mapping entries to specified column names
         Args:
-            bucket : the S3 Bucket
+            region: AWS region where the bucket is located
```

Comment:
Is this method only intended to non-local use. If so, should we say so here (I noticed you had the local/non-local distinction in a few other docstrings, which is very helpful).